runtests

```diff
@@ -17,6 +21,8 @@ Usage: ${0##*/} [-f CMDFILES] [-c CMDLINE] [-o LOGFILE]
 Examples:
   ./runtests -f cet/tests
   ./runtests -f cet/tests -o cet_tests.log
+  ./runtests -f tests-client  // exeucte all tests-client or tests in subfolders
+  ./runtests -f tests-server  // exeucte all tests-server or tests in subfolders
```

Comment:
Missing folder 'cet' before the tests file?

Reply:
It's for all subfolders tests, not only for cet subfolder.

Thanks!